sklearn/decomposition/kernel_pca.py

```diff
@@ -81,6 +81,11 @@ class KernelPCA(BaseEstimator, TransformerMixin):
         A pseudo random number generator used for the initialization of the
         residuals when eigen_solver == 'arpack'.
 
+    copy : boolean, default=True
```

Comment:
@agramfort I had the same thought and did a code search. Linear models use `copy_X` mostly and decompositions use `copy`, just like @toastedcornflakes noted.  (So it's not really a question of fragmentation, both are used.)

I guess it makes some sense, since here there is little ambiguity as to what will be copied.